.github/workflows/build_and_test_workflow.yml

```diff
@@ -119,7 +66,12 @@ jobs:
           npm i -g yarn@1.22.10
           yarn config set network-timeout 1000000 -g
 
-      - name: Configure Yarn Cache
+      - name: Configure Yarn Cache (Linux)
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
In prep for `darwin`, i think we should just do `if: matrix.os != 'windows-latest'`. What do you think?

Reply:
Should it be the case for every `if` condition in this file ?

Reply:
yeah; thanks.